Requirements/ViewModels/RequirementBrowser/Rows/RequirementsGroupRowViewModel.cs

```diff
@@ -11,154 +30,93 @@ namespace CDP4Requirements.ViewModels
-            base.UpdateProperties();
-
-            this.UpdateRequirementGroupRows();
-            this.UpdateIsDeprecatedDerivedFromContainerRequirementsSpecification();
-        }
-
-        /// <summary>
-        /// Initializes the subscriptions
-        /// </summary>
-        protected override void InitializeSubscriptions()
-        {
-            base.InitializeSubscriptions();
-
-            var requirementsSpecificationRowViewModel =
-                this.ContainerViewModel as RequirementsSpecificationRowViewModel;
-            if (requirementsSpecificationRowViewModel != null)
-            {
-                var containerIsDeprecatedSubscription =
-                    requirementsSpecificationRowViewModel.WhenAnyValue(vm => vm.IsDeprecated)
-                        .Subscribe(_ => this.UpdateIsDeprecatedDerivedFromContainerRequirementsSpecification());
-                this.Disposables.Add(containerIsDeprecatedSubscription);
-            }
-        }
-
-        /// <summary>
-        /// Updates the IsDeprecated property based on the value of the container <see cref="RequirementsSpecification"/>
+        /// Gets the categories name
         /// </summary>
-        private void UpdateIsDeprecatedDerivedFromContainerRequirementsSpecification()
+        public string Categories
         {
-            var requirementsSpecification = this.Thing.GetContainerOfType<RequirementsSpecification>();
-            if (requirementsSpecification != null)
+            get
             {
-                this.IsDeprecated = requirementsSpecification.IsDeprecated;
-
-                var groupRows = this.ContainedRows.OfType<RequirementsGroupRowViewModel>();
-                this.RecursivelyUpdateIsDeprecated(groupRows, requirementsSpecification.IsDeprecated);
-            }
-        }
-
-        /// <summary>
-        /// Recursively Update the IsDeprecated property
-        /// </summary>
-        /// <param name="groupRows">
-        /// The collection of rows in the group.
-        /// </param>
-        /// <param name="isDeprecated">The deprecated state.</param>
-        private void RecursivelyUpdateIsDeprecated(IEnumerable<RequirementsGroupRowViewModel> groupRows,
-            bool isDeprecated)
-        {
-            foreach (var requirementsGroupRowViewModel in groupRows)
-            {
-                requirementsGroupRowViewModel.IsDeprecated = isDeprecated;
+                if (this.Thing == null)
+                {
+                    return string.Empty;
+                }
 
-                var containedGroupRows =
-                    requirementsGroupRowViewModel.ContainedRows.OfType<RequirementsGroupRowViewModel>();
-                requirementsGroupRowViewModel.RecursivelyUpdateIsDeprecated(containedGroupRows, isDeprecated);
+                var firstOrDefault = this.Thing.Category.FirstOrDefault();
+                return firstOrDefault != null ? string.Join(", ", this.Thing.Category.Select(x => x.Name)) : string.Empty;
```

Comment:
this is a bit strange. Why not use:

`this.Thing.Category.Any() ? string.Join(", ", this.Thing.Category.Select(x => x.Name)) : string.Empty;`

then you don't need the firstordefault.